test/Mono.Linker.Tests.Cases/DataFlow/TypeDataflow.cs

```diff
@@ -0,0 +1,127 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
All the cases inside here all already covered in ObjectGetType tests so it's safe to delete this file